packages/cardpay-sdk/sdk/safe-module.ts

```diff
@@ -577,4 +576,9 @@ export default abstract class SafeModule {
       expectedModuleAddress: enableModuleTxs.expectedModuleAddress,
     };
   }
+
+  async isModuleEnabled(safeAddress: string, moduleAddress: string): Promise<boolean> {
```

Comment:
This checks if a particular moduleAddress is enabled on a safe. I found this function not particularly useful because a safe doesn't share modules with other safes. But its a good way to confirm that you have installed a module given your expectedModuleAddress